packages/client/src/components/DisplayCard.component.tsx

```diff
@@ -0,0 +1,29 @@
+import Card from '@mui/material/Card';
+import CardContent from '@mui/material/CardContent';
+import Typography from '@mui/material/Typography';
+import { WorkSuitIcon } from '@components/images/WorkSuitIcon';
+import { RestIcon } from '@components/images/RestIcon';
+
+type DisplayCardProps = {
```

Comment:
Sometimes you use type and sometimes you use interfaces, it is best to be consistent. I would suggest interfaces